client/sites/settings/site/visibility/index.jsx

```diff
@@ -152,7 +150,9 @@ const LaunchSite = () => {
 	} else {
 		btnComponent = (
 			<Button
-				href={ `/start/launch-site?siteSlug=${ siteSlug }&source=general-settings&new=${ site.title }&search=yes` }
```

Comment:
I'm just testing what the `new` and `search` query params do. If they're important perhaps they should be in the redux action too.

Reply:
Yes, the `new` query param is important. It pre-fills the domain search box

Reply:
I am guessing they are to do with the populating the domain search maybe?

Reply:
Passing these through now - doesn't seem to result in a domain search though, but apparently there is a separate bug in trunk already with this